tutor/specs/e2e/add-edit-question.e2e.js

```diff
@@ -54,9 +53,10 @@ describe('Add/Edit Questions', () => {
         await page.type(editorSel, 'Hello World!')
         await page.click('.two-step-info') // really just clicking outside editor
         await page.click('testEl=publish-btn')
+        await page.waitForTimeout(1000)
```

Comment:
This is pretty long but is needed for the Publish action, otherwise the user menu will not be available for the next beforeEach run logout action.